scripts/configure-hypervisor.sh

```diff
@@ -326,17 +326,14 @@ generate_qemu_options() {
 	# implicitly enabled in Fedora 27).
 	qemu_options+=(size:--disable-linux-aio)
 
-	if [[ "${qemu_version_major}" -ge 4 || ( "${qemu_version_major}" -eq 3  &&  "${qemu_version_minor}" -ge 1 ) ]]; then
+	if [[ "${qemu_version_major}" -ge 4 || ( "${qemu_version_major}" -eq 3  &&  "${qemu_version_minor}" -ge 0 ) ]]; then
```

Comment:
thanks, I think now we can simplify this, how about `if [[  "${qemu_version_major}" -ge 3 ]]`?

Reply:
sure